assets/js/translator.js

```diff
@@ -133,8 +133,6 @@ if(modeEnabled('translation')) {
             synchronizeTextareaHeights();
         });
 
-        $(window).resize(synchronizeTextareaHeights);
```

Comment:
Um, the commit message says - `Improve textarea mechanics`.
The link to the blame where this line of code was added it [link](https://github.com/goavki/apertium-html-tools/blame/2e3cc23121697bf7f3eb26531e090791d649227f/assets/js/translator.js)